addons/mrp/tests/test_stock_report.py

```diff
@@ -467,3 +467,68 @@ def test_mo_overview_same_component(self):
         self.assertEqual(len(repl1), 1)
         self.assertEqual(repl0[0]['summary']['quantity'], 3)
         self.assertEqual(repl1[0]['summary']['quantity'], 5)
+
+    def test_mrp_report_overview(self):
+        product_parent = self.env['product.product'].create({
+            'name': 'Choco Cake',
+            'type': 'product',
+            'route_ids': [Command.set([self.ref('mrp.route_warehouse0_manufacture')])],
+        })
+        product_child = self.env['product.product'].create({
+            'name': 'Double Choco Cake',
+            'type': 'product',
+            'route_ids': [Command.set([self.ref('mrp.route_warehouse0_manufacture')])],
+        })
```

Comment:
When dealing with data in a test
\- If possible, always use the ones created in the setup
\- Else, always create them in batch